components/TxForm.tsx

```diff
@@ -137,9 +137,9 @@ export const TxForm = ({ recipientAddress }: { recipientAddress?: string }) => {
           <Input
             variant="outline"
             size="lg"
-            pr="4.5rem"
             height="4rem"
             mb=".25rem"
+            placeholder="0zk1qyn0qa5rgk7z2l8wyncpynmy..."
```

Comment:
We can display a whole address if we switch to a textarea, but the form will look a little weirder